KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -104,12 +117,13 @@ public GamePanel(GameFrame frame) {
         // Add the mouse listener that calls the mouse click event handler
         addMouseListener(new MouseAdapter() {
             /**
-             * Triggered when the user clicks on the game panel.
-             * Calls the game panel's click event handler.
+             * Triggered when the user clicks on the game panel. Calls the game
+             * panel's click event handler.
+             *
              * @param evt The event representing the mouse click
              */
             @Override
-            public void mouseClicked(MouseEvent evt) {
+            public final void mouseClicked(MouseEvent evt) {
```

Comment:
does this have to be final? I would feel better if it wasn't

Reply:
Sure, it doesnt have to be final
its only like that because of the warning we triedto fix last night